src/vstart.sh

```diff
@@ -671,6 +671,20 @@ EOF
         run 'mgr' $CEPH_BIN/ceph-mgr -i $name $ARGS
     done
 
+    wait_active_start=`date +%s`
+    while [ 1 ]; do
+      if ceph_adm -s 2>&1 | grep 'mgr:.*\?(active)' >&/dev/null; then
+        break
+      fi
+      echo "Waiting for MGR to become active."
+      waited_for=$((`date +%s` - wait_active_start))
```

Comment:
yeah, i'll rework the patch. tbh, this was a nasty quick kludge to get my day starting ;)